src/Cmfd.h

```diff
@@ -365,7 +365,7 @@ class Cmfd {
   double getDistanceToCentroid(Point* centroid, int cell_id,
                                      int stencil_index);
   void getSurfaceDiffusionCoefficient(int cmfd_cell, int surface,
-        int group, int moc_iteration,double& dif_surf, double& dif_surf_corr);
+        int group, int moc_iteration, CMFD_PRECISION& dif_surf, CMFD_PRECISION& dif_surf_corr);
```

Comment:
line too long

Reply:
Fixed. It's part of the previous PR to fix this line